### PR TITLE
Add Feb 2024 newsletter

### PR DESCRIPTION
**Pull request by ericholscher**
This swaps this content over for our website blog,
and makes that the official place for writing new blog content!

* Blog post: https://read-the-docs-website--247.com.readthedocs.build/blog/2024/01/read-the-docs-newsletter-february-2024/
* Atom feed: https://read-the-docs-website--247.com.readthedocs.build/blog/atom.xml

----
📚 Documentation preview 📚: https://read-the-docs-website--247.com.readthedocs.build/

### CODE REVIEW

**Comment by agjohnson** [comment]
I thought there might be some bugs with pages like:

https://read-the-docs-website--247.com.readthedocs.build/blog/category/newsletter.html

But I think the issue was just missing content actually, the category page looks to generate correctly with proper posts (instead of drafts, when I tested things).

File: content/posts/newsletter-february-2024.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Possible issues
+
+- We are investigating [removing support for VCS systems other than Git](https://github.com/readthedocs/readthedocs.org/issues/8840). We have sent an email to all users who are using other VCS systems for feedback as a first step.
+- We have [defaulted traffic analytics off](https://github.com/readthedocs/readthedocs.org/pull/11056) for users of our new Addons. This default is to save on resources for projects that aren't looking at our analytics data.
+- Our [beta dashboard](https://beta.readthedocs.org/) continues to be tested in public beta, and new functionality for Addons configuration is only be available in that new interface.
+
+-----
+
+Questions? Comments? Ideas for the next newsletter? [Contact us](mailto:hello@readthedocs.org)!
```
Comment (agjohnson) [inline content/posts/newsletter-february-2024.md:32]: We could move this to the template eventually too, there are two sort of callouts at the bottom. Not important for this PR though.

File: pelicanconf.py [hunk 2]
```diff
@@ -38,13 +37,12 @@
 
 # Feed (RSS/Atom) settings
 FEED_DOMAIN = SITEURL
-FEED_RSS = FEED_RSS_URL = "blog/feed.rss"
-FEED_ATOM = FEED_ATOM_URL = "blog/atom.xml"
+FEED_ATOM = "feeds/atom.xml"
+FEED_RSS = "feeds/feed.rss"
```
Comment (agjohnson) [inline pelicanconf.py:41]: I want to double check on the base template too, I think the base template might have hardcoded the wrong URLs in at some point or something, as there are 2 extra feeds listed in the head.

**Review by agjohnson — APPROVED** [review]
The content looks quite a bit better on the blog here :+1:

File: content/posts/newsletter-february-2024.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community, build-in-public
+authors: Eric Holscher
+status: published
+
+## News and updates
+
+* We moved our new blog posts to be published on our website, so going forward you can find it [directly on our website](https://about.readthedocs.org/blog/).
+* We shipped an update that allows [Sphinx builds to share environments](https://github.com/readthedocs/readthedocs.org/pull/11073). This should make builds of multiple formats faster.
```
Comment (humitos) [inline content/posts/newsletter-february-2024.md:12]: Shouldn't we use sentence break here?

File: content/posts/newsletter-february-2024.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+You can always see the latest changes to our platforms in our [changelog 📃](https://docs.readthedocs.io/page/changelog.html).
+
+## Upcoming changes
+
+- We continue to work on improving Addons. Our goal is to default all projects to using Addons in Q2 of 2024. We are working on improving the user experience, and adding more Addons to make the transition easier.
+
+Want to follow along with our development progress? View our [development roadmap 📍️](https://github.com/orgs/readthedocs/projects/156/views/1)
+
+## Possible issues
```
Comment (stsewd) [inline content/posts/newsletter-february-2024.md:24]: Maybe mention that users should update their readthedocs-sphinx-search extension? We also emailed users about it https://github.com/readthedocs/readthedocs-sphinx-search/security/advisories/GHSA-xgfm-fjx6-62mj.

File: content/posts/newsletter-february-2024.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+title: Read the Docs Newsletter - February 2024
+date: Jan 1, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community, build-in-public
+authors: Eric Holscher
+status: published
+
+## News and updates
```
Comment (stsewd) [inline content/posts/newsletter-february-2024.md:9]: Forced redirects are now available for everyone, maybe mention the new limits as well?

File: content/posts/newsletter-february-2024.md (hunk 3 above)
Comment (ericholscher) [reply content/posts/newsletter-february-2024.md:12]: Sentence breaks are hard to do in list items with indention, so not super worried about it.

File: content/posts/newsletter-february-2024.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+title: Read the Docs Newsletter - February 2024
+date: Jan 1, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community, build-in-public
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply content/posts/newsletter-february-2024.md:5]: https://buildinpublic.xyz/what-is-build-in-public

[the remark this answers is not in the document]
Comment (ericholscher) [reply content/posts/newsletter-february-2024.md:5]: I copied it over from ads where we share revenue numbers, so probably doesn't apply as much here.

**Comment by ericholscher** [comment]
Feedback should mostly be resolved, and ready to go live tomorrow.

File: readthedocs_theme/templates/includes/topnav.html [hunk 7]
```diff
@@ -1,6 +1,7 @@
 {% macro menu_main() %}
   <a class="item{% if page and page.slug == 'features' %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/features/">Features</a>
   <a class="item{% if page and page.slug == 'pricing' %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/pricing/">Pricing</a>
+  <a class="item{% if page and 'blog' in page.slug %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/blog/">Blog</a>
```
Comment (agjohnson) [inline readthedocs_theme/templates/includes/topnav.html:4]: This is okay for now, but it also doesn't feel like a great top level menu item. I'd like to revisit re-re-naming the "Features" and turning that into a dropdown menu to help reduce bloat on the top menu.

File: content/posts/newsletter-february-2024.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+authors: Eric Holscher
+status: published
+
+## News and updates
+
+* Forced redirects are now available for all users, but redirects are limited based on plan types. This is an update from our previous newsletter that comes from our redirects refactor.
+* We moved our new blog posts to be published on our website, so going forward you can find it [directly on our website](https://about.readthedocs.org/blog/).
+* We shipped an update that allows [Sphinx builds to share environments](https://github.com/readthedocs/readthedocs.org/pull/11073). This should make builds of multiple formats faster.
+* We shipped a few different security fixes that were found by Santos Gallegos, a member of our team. You can read more about them in our [security advisories](https://github.com/readthedocs/readthedocs.org/security/advisories)
+* We shipped an [updated settings page for Addons](https://github.com/readthedocs/readthedocs.org/pull/11031) in our new beta dashboard. This allows enabling and disabling each addon.
```
Comment (humitos) [inline content/posts/newsletter-february-2024.md:15]:
```suggestion
* We shipped an [updated settings page for Addons](https://github.com/readthedocs/readthedocs.org/pull/11031) in our new beta dashboard. This allows enabling and disabling each addon.
* We updated our `build.tools` versions to use the Node.js to its latest LTS version (`v20.11.0) and added support for Go 1.21 and Rust 1.75.
```

**Review by humitos — APPROVED** [review]

File: readthedocs_theme/templates/includes/topnav.html (hunk 7 above)
Comment (ericholscher) [reply readthedocs_theme/templates/includes/topnav.html:4]: Yea -- would be nice to have `Resources` or similar as a top level with `Blog`, `Docs`, `Support` in there probably.

**Pull request merged**